src/modding/NewModGUI.tscn

```diff
@@ -1,9 +1,10 @@
 [gd_scene load_steps=4 format=2]
 
-[ext_resource path="res://src/gui_common/CustomDialog.tscn" type="PackedScene" id=1]
+[ext_resource path="res://src/gui_common/dialogs/CustomDialog.tscn" type="PackedScene" id=1]
 [ext_resource path="res://src/modding/NewModGUI.cs" type="Script" id=2]
 [ext_resource path="res://src/gui_common/fonts/Lato-Bold-AlmostSmaller.tres" type="DynamicFont" id=3]
 
+
```

Comment:
A bunch of extra newlines seem to get inserted. For me Godot doesn't add these in the scene files...

Reply:
I don't know, these were automatic. I noticed I'm running 3.5, latest is 3.5.1 right? miiight the cause. Should these be removed now or let them be?

Reply:
We are still using 3.5. Removing these would reduce the number of translation file changes *a lot*.